demosplan/DemosPlanProcedureBundle/Logic/PrepareReportFromProcedureService.php

```diff
@@ -307,12 +304,33 @@ public function createReportEntry(Procedure $sourceProcedure, Procedure $destina
     }
 
     /**
-     * @param User|null $user avoid passing `null`, as we want to show a user when the report is listed in the UI
+     * @return User|non-empty-string
+     */
+    private function getUserForReportEntry(): User|string
+    {
+        $user = null;
+        try {
+            $user = $this->currentUser->getUser();
+        } catch (UserNotFoundException $e) {
+            $this->logger->info('No user found for report entry creation, falling back to default.', [$e]);
+        }
+        if (null !== $user && '' === $user->getFullname()) {
```

Comment:
Don't you mean `&& '' !== $user->getFullname()`? I thought the problem was exactly when the user name is an empty string

Reply:
Yes, thank you. I switched the condition around a bit too often. 665ea712cd2afee1740afc358f39f6633824f36e